source/jormungandr/tests/kirin_realtime_tests.py

```diff
@@ -127,8 +129,8 @@ def _check_train_cancel_disruption(dis):
         assert vjs[0]['id'] == 'vjA'
 
         new_response = self.query_region(journey_basic_query + "&data_freshness=realtime")
-        assert get_arrivals(new_response) == ['20120614T080436', '20120614T180222']
-        assert get_used_vj(new_response) == [[], ['vjB']]
+        assert set(get_arrivals(new_response)) == set(['20120614T080436', '20120614T080223'])
+        assert get_used_vj(new_response) == [['vjM'], []]
```

Comment:
if you want the 'unordered-ness ', you can use tuple which is hashable in python

Reply:
ok

Reply:
just for fun, you can take a look at frozenset